templates/rhdm75-authoring-ha.yaml

```diff
@@ -173,12 +173,17 @@ parameters:
 - displayName: DataGrid Image
   description: DataGrid image.
   name: DATAGRID_IMAGE
-  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.1
+  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.2
+  required: true
+- displayName: DataGrid Container CPU Limit
+  description: DataGrid Container cpu limit.
+  name: DATAGRID_CPU_LIMIT
+  value: "4"
   required: true
 - displayName: DataGrid Container Memory Limit
   description: DataGrid Container memory limit.
   name: DATAGRID_MEMORY_LIMIT
-  value: "512Mi"
+  value: "8Gi"
```

Comment:
Datagrid requires 8 GB of memory and 4 CPU? Isn't it too much? From the JIRA description it looks more like a JVM configuration issue (JVM allocates much less memory than available in pod).

Reply:
@sutaakar I would hope it doesn't need that much... I was just putting in what I saw with @jakubschwan 's workaround on the jira. Since the memory and cpu are both configurable, would do you guys think is a sensible default? 1Gi? 2Gi? How many cpu's... 1? 2? (previously it was set to 0.5)...

Reply:
@errantepiphany @sutaakar recommended value in this case (for DataGrid) should be 4 CPU and 4 Gi of memory. 1Gi is imo not enough as for JVM would be used 50% and we actually saw OOM issues with set -Xmx512m to the DataGird.

Reply:
@sutaakar memory from container is allocated as expected by cct_module scripts, there is env. variable to set jvm memory usage in percents, so -Xmx is set by JAVA_MAX_MEM_RATIO - default value is 50

Reply:
ok

Reply:
Changed to 4Gi.

Reply:
4Gi I still believe that it is too much, I don't think JDG requires more than 1Gi though.